### PR TITLE
SIP-2010: Add ARB market to PerpsV2

### PR DESCRIPTION
**Pull request by leomassazza**
Implements https://sips.synthetix.io/sips/sip-2010/

### CODE REVIEW

File: publish/assets.json [hunk 1]
```diff
@@ -494,5 +494,11 @@
 		"category": "crypto",
 		"sign": "",
 		"description": "NEAR"
+	},
+	"ARB": {
+		"asset": "ARB",
+		"category": "crypto",
+		"sign": "",
+		"description": "ARB"
```
Comment (kaleb-keny) [inline publish/assets.json:502]: For the sake of consistency, would it be ok to update the  description here to "Arbitrum"

File: publish/deployed/mainnet-ovm/perpsv2-markets.json [hunk 2]
```diff
@@ -596,5 +596,31 @@
 		"offchainMarketKey": "ocGBPPERP",
 		"paused": false,
 		"offchainPaused": false
+	},
+	{
+		"marketKey": "sARBPERP",
+		"asset": "ARB",
```
Comment (kaleb-keny) [inline publish/deployed/mainnet-ovm/perpsv2-markets.json:602]: I believe this what the line that troy was asking about, if we can drop the `s` for, maybe there was a misunderstanding when we spoke about it, as `marketKey` has the `s`, but `asset` doesn't need one.

File: publish/deployed/mainnet-ovm/perpsv2-markets.json [hunk 3]
```diff
@@ -596,5 +596,31 @@
...
+		"makerFee": "0.05",
+		"takerFee": "0.05",
+		"overrideCommitFee": "0",
+		"takerFeeDelayedOrder": "0.05",
+		"makerFeeDelayedOrder": "0.05",
+		"takerFeeOffchainDelayedOrder": "0.0004",
+		"makerFeeOffchainDelayedOrder": "0.0012",
```
Comment (kaleb-keny) [inline publish/deployed/mainnet-ovm/perpsv2-markets.json:609]: I update these two troy pointed it out yesterday, makerFee is almost always <= takerFee, in order to encourage arbitrage and a balanced skew. If you can please flip these, I posted the necessary PR on the SIP site, that is yet to be merged (i.e. makerFee = 4 bp / takerFee = 12 bp)

**Review by kaleb-keny — CHANGES_REQUESTED** [review]

File: publish/assets.json (hunk 1 above)
Comment (leomassazza) [reply publish/assets.json]: Fixed

File: publish/deployed/mainnet-ovm/perpsv2-markets.json (hunk 3 above)
Comment (leomassazza) [reply publish/deployed/mainnet-ovm/perpsv2-markets.json]: Fixed

**Review by kaleb-keny — APPROVED** [review]

**Review by barrasso — APPROVED** [review]

**Review by kaleb-keny — APPROVED** [review]

**Pull request merged**